pyproject.toml

```diff
@@ -12,6 +12,9 @@ classifiers = [
     'Intended Audience :: Developers',
     'Programming Language :: Python :: 3.6',
     'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
```

Comment:
I think we can remove those versions no? We only support 3.7.

Reply:
No, this can run remotely, and we connect to the remote redis in the README example. If anything we should add more, troves. Am I wrong?

Reply:
This runs remotely but I believe it will not work on python 3.8 due to binary incompatibility

Reply:
Ah.. okay, will remove the trove.